### ISSUE TITLE
Console output issue when no retype config is found

### ISSUE THREAD

**Issue by fabriciomurta** [retypeapp/action-build#17]
When the action wires up default config because it doesn't find one in the repo, the output does not feed a line break and the messages' readability becomes compromised.

Affected output:

```
Setting up build arguments: initialize default configuration::group::Command: retype init --verbose
```

**Comment by fabriciomurta** [comment]
Fixed via  b8eba2b3febf28fd54ac9f15176d29a4e9064d49.

**Comment by fabriciomurta** [comment]
Output when no configuration found:
![image](https://user-images.githubusercontent.com/10730372/143510092-f9bd2a2e-e8e7-4e17-820a-0f78a0f71f19.png)

Output when `retype.yml` is found at root:
![image](https://user-images.githubusercontent.com/10730372/143510123-85513437-52ba-40f8-86c2-42a7101d3faf.png)
